modules/rostests/kmtests/ntos_io/IoFilesystem.c

```diff
@@ -365,6 +428,61 @@ TestRelativeNames(VOID)
         { L"C:",                        L"\\",                              TRUE,   STATUS_OBJECT_PATH_NOT_FOUND },
         { L"C:",                        L"file",                            TRUE,   STATUS_OBJECT_PATH_NOT_FOUND },
     };
+
+    RelativeNameTest Tests_NTFS[] =
```

Comment:
I think it would be nicer to have one table and extra fields, like 
- duplicate the IsDirectory and Status field
- and/or replace IsDirectory with flags: FALSE (0), TRUE (1), FAT32_TRUE (2), NTFS_TRUE (4)
- and/or replace Status with flags: FL_SUCCESS (0), FL_INVALID_FAT32 (1), FL_INVALID_NTFS (2), FL_INVALID (3), ...

This would make it more obvious where things are FS dependent.